api-client-util/src/main/java/com/takipi/api/client/util/regression/DeploymentTimespan.java

```diff
@@ -0,0 +1,25 @@
+package com.takipi.api.client.util.regression;
+
+import com.takipi.common.util.Pair;
+import org.joda.time.DateTime;
+
+import java.util.Map;
+
+public class DeploymentTimespan {
```

Comment:
Class and data member field names should show that this class holds multiple deployments